build/ivysettings.xml

```diff
@@ -20,11 +20,18 @@ SPDX-License-Identifier: Apache-2.0
     <property name="repo.dir" value="${basedir}/repository"/>
     <resolvers>
         <!-- Resolver for downloading dependencies -->
-        <url name="Sosy-Lab" descriptor="required">
-            <ivy pattern="${ivy.repo.url}/[organisation]/[module]/ivy-[revision].xml" />
-            <artifact pattern="${ivy.repo.url}/[organisation]/[module]/[artifact]-[revision](-[classifier]).[ext]" />
-        </url>
-
+        <chain name="Sosy-Lab">
+            <url name="Sosy-Labs" descriptor="required">
+                <ivy pattern="${ivy.repo.url}/[organisation]/[module]/ivy-[revision].xml" />
+                <artifact pattern="${ivy.repo.url}/[organisation]/[module]/[artifact]-[revision](-[classifier]).[ext]" />
+            </url>
+            
+            <filesystem name="Sosy-Hacks" descriptor="required">
```

Comment:
You could also provide a file `build.properties` with the following content:
```
# ivy.repo.url=https://www.sosy-lab.org/ivy
ivy.repo.url=file:/home/user/workspace/java-smt/repository
```

Then Ant/Ivy would use that directory as repository.

Reply:
Hello Karlheinz,
unfortunately the remote repository is still needed to get the java dependencies for the build.

I can revert the first commit from my branch once the code is ready to be merged. This should get everything back in line with the main branch